manifests/plugin/default_hostgroup.pp

```diff
@@ -1,4 +1,20 @@
-# Installs foreman_default_hostgroup plugin
-class foreman::plugin::default_hostgroup {
-  foreman::plugin {'default_hostgroup': }
+# = Foreman Default Hostgroup
+#
+# This class installs the default_hostgroup plugin and optionally manages the configuration file
+#
+# === Parameters:
+#
+# $hostgroups:: An hash of hostgroup names and facts to add to the configuration
```

Comment:
For simple documentation (when there are no parameter groups) we prefer puppet-strings' yardoc format:

```suggestion
# @summary This class installs the default_hostgroup plugin and optionally manages the configuration file
#
# @param hostgroups An hash of hostgroup names and facts to add to the configuration
```